core/sql/arkcmp/CompException.cpp

```diff
@@ -113,11 +114,21 @@ FatalException::FatalException(const char *msg,
   else {
     msg_[0] = 0;
   }
+  if(stackTrace) {
+    strncpy(stackTrace_, stackTrace, sizeof(stackTrace_));
+    stackTrace_[sizeof(stackTrace_)-1] = 0;
```

Comment:
Yes. stackTrace_ and msg_ are declared as stackTrace_[max len] and msg_[max len] in core/sql/arkcmp/CompException.h.  This is part of the diffs in this PR.